pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -716,8 +717,14 @@ func (m *kubeGenericRuntimeManager) killContainer(pod *v1.Pod, containerID kubec
 
 	err := m.runtimeService.StopContainer(containerID.ID, gracePeriod)
 	if err != nil {
-		klog.ErrorS(err, "Container termination failed with gracePeriod", "pod", klog.KObj(pod), "podUID", pod.UID,
+		if !libdocker.IsContainerNotFoundError(err){
```

Comment:
we should support all container runtimes, not just docker

Reply:
agree with you. there need one specification for error message returned by different runtime or cri component in the future. In actual, there are some other places use libdocker's func also. They should be considered together.

Reply:
resolved